cloudfoundry-operations/src/test/java/org/cloudfoundry/operations/domains/DefaultDomainsTest.java

```diff
@@ -78,6 +84,38 @@ private static void requestOrganizationsEmpty(CloudFoundryClient cloudFoundryCli
                     .build()));
     }
 
+    private static void requestDomains(CloudFoundryClient cloudFoundryClient, String name, String ownerId) {
```

Comment:
@LokeshN When the mainline code is refactored with the two network requests, these methods should match the newly introduced method names in the main code.  This mapping between the two makes it easy for us to identify network requests that need to be mocked out.